conf/scalardb-graphql-custom-values.yaml

```diff
@@ -0,0 +1,53 @@
+replicaCount: 3
+
+scalarDbGraphQlConfiguration:
+  #
+  # DynamoDB
+  #
+  # contactPoints: <REGION>
+  # username: <AWS_ACCESS_KEY_ID>
+  # password: <AWS_ACCESS_SECRET_KEY>
+  # storage: dynamo
+
+  logLevel: INFO
+  path: /graphql
+  namespaces: "<comma-separated namespace list>"
+  graphiql: "true"
+
+ingress:
+  enabled: true
+  className: alb
+  annotations:
+    # ALB
+    alb.ingress.kubernetes.io/scheme: internal
+    alb.ingress.kubernetes.io/target-group-attributes: stickiness.enabled=true,stickiness.lb_cookie.duration_seconds=60
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/healthcheck-path: /graphql?query=%7B__typename%7D
+  hosts:
+    - host: ""
+      paths:
+        - path: /graphql
+          pathType: Exact
```

Comment:
`pathType: Exact` is enough for the GraphQL single endpoint. And I confirmed this works with ALB.
https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types

<img width="1235" alt="Screen Shot 2022-07-01 at 20 10 48" src="https://user-images.githubusercontent.com/11030/176886185-64bc97e9-a7ba-4a45-867e-794ea3504d17.png">